README.md

```diff
@@ -9,25 +9,40 @@ The following steps will install the `tigerbeetle-node` module to your current w
 * NodeJS >= `14.0.0`. _(If the correct version is not installed, an installation error will occur)_ 
 
 > Your operating system should be Linux (kernel >= v5.6) or macOS. Windows support is not yet available but is in the works.
-     
+
 ### YARN Package Manager
 
-```sh
-# Run the following from this directory
-yarn add
+```shell
+# Run the following from this directory:
+yarn postinstall #Download NodeJS Headers, Install Zig, Build TigerBeetle
+yarn && yarn build && yarn add
 ```
 or
 
-### NPM Package Manager 
-```sh
-npm install tigerbeetle-node
+### NPM Package Manager
+Run the following command from this directory:
+```shell
+npm install .
+```
+
+**Yarn - Run Test**
+Ensure TigerBeetle (`init` & `start`) is running on the port configured in `test.ts`, then run: 
+```shell
+./tigerbeetle init --cluster=1 --replica=0 --directory=.
+./tigerbeetle start --cluster=1 --replica=0 --directory=. --addresses=3001
+yarn test
+```
+
+**Yarn - Run Benchmark**
+Run the benchmark (The `benchmark` will automatically start TigerBeetle on port `3001` _(single replica)_:
+```shell
+yarn benchmark
```

Comment:
Might be better to group this with the development stuff below. That way we can separate it for people just wanting to install and people wanting to develop. What do you think?

Reply:
I agree :100: I have updated the `README.md`.